src/config.js

```diff
@@ -23,7 +23,19 @@ const Config = {
             }
         },
         apis: ["src/router/handlers/*.js"]
-    }
+    },
+    userPublicKey: process.env.USER_PUBLIC_KEY,
+    userPrivateKey: process.env.USER_PRIVATE_KEY,
+    adminPublicKey: process.env.ADMIN_PUBLIC_KEY,
+    adminPrivateKey: process.env.ADMIN_PRIVATE_KEY,
```

Comment:
I like how you're signing it with a different key depending on the type of user—this is quite nifty!